### PR TITLE
CAPSLib loading & macOS improvements

### PR DESCRIPTION
**Pull request by renaudguerin**
Salut Jean-François,

In this PR I tried to improve the reliability of autodetecting / loading the CAPS library (especially on macOS), along with a few other minor improvements. 

1. Added support for having several values (comma-separated) in `SPSCAPS_LIB_NAME`.
It will try each one in order (I've added more detailed logging too), and I have changed the default values of the variable in Linux/macOS to take advantage of this.
This is similar to what [other tools are doing](https://github.com/keirf/greaseweazle/blob/8d5080d2929d6a4be46f00ec370152cde1705111/src/greaseweazle/image/ipf.py#L296) with CAPSlib, and it should better autodetect the [different](https://fs-uae.net/download#macosx) [builds](https://github.com/renaudguerin/capsimage/releases/tag/v5.1) that are floating around.
On macOS, it means users should copy `CAPSImage.framework` or `CAPSImg.framework` to `/Library/Frameworks` (admin access needed) and HxC will use either of them. It should address issues reported [here](https://github.com/jfdelnero/HxCFloppyEmulator/issues/7) or [here](https://torlus.com/floppy/forum/viewtopic.php?f=5&t=3760).
2. Slightly improved the Mac bundle creation script. Adding `CFBundleIdentifier` to the plist should also fix Spotlight not finding the app.
3. Some other minor tweaks to `.gitignore` (it didn't cover all of fltk when building on Mac) and README.

Just built & tested on macOS Ventura 13.5. I didn't test the Windows logging addition, hopefully it works !

### CODE REVIEW

File: build/maccreatebundle [hunk 1]
```diff
@@ -1,4 +1,5 @@
-export VERSION=`cat version.h | grep "define STR_FILE_VERSION2" | sed 's/.*STR_FILE_VERSION2 //' `
+#!/usr/bin/env bash
+export VERSION=`grep "define STR_FILE_VERSION2" version.h | cut -f2 -d\"`
```
Comment (renaudguerin) [inline build/maccreatebundle:2]: This also removes the double quotes from the VERSION string : they seemed to cause a malformed plist file.

File: HxCFloppyEmulator_software/sources/thirdpartylibs/fltk/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+fltk-1.x.x*
```
Comment (renaudguerin) [inline HxCFloppyEmulator_software/sources/thirdpartylibs/fltk/.gitignore:1]: Also covers the `fltk-1.x.x-arm64/` / `fltk-1.x.x-x86_64/` directories I had on my Mac after building

File: libhxcfe/sources/floppy_loader.c [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /*
 //
-// Copyright (C) 2006-2023 Jean-Fran�ois DEL NERO
+// Copyright (C) 2006-2023 Jean-François DEL NERO
```
Comment (renaudguerin) [inline libhxcfe/sources/floppy_loader.c:3]: Encoding problems after I edited / saved the files. Maybe your original source wasn't UTF-8 or there was an issue in the SF -> Github migration ?

**Comment by jfdelnero** [comment]
Thanks. I will send you the build to test it.

**Pull request merged**